libraries/lib-datahandler/src/main/thrift/codescoop.thrift

```diff
@@ -51,7 +51,7 @@ struct CodescoopComponentOrigin {
 }
 
 struct CodescoopSearchFilter {
-    1: required list<i32> type;
+    1: required list<string> type;
```

Comment:
I suppose nothing is stored of the thrift requests? So this change should not be a problem.

If older exchanged objects would be persisted somewhere, this might lead to problems if they should be exchanged again.